Marlin/language_fr.h

```diff
@@ -193,6 +193,7 @@
 #define MSG_INFO_PROTOCOL                   _UxGT("Protocole")
 #define MSG_LIGHTS_ON                       _UxGT("Allumer boitier")
 #define MSG_LIGHTS_OFF                      _UxGT("Eteindre boitier")
+#define MSG_LIGHTS_NONE                     _UxGT("Pas de lumiere boitier")
```

Comment:
This message should be defined in 'language.h' because it is sent over USB/serial.
Only 'displayed' messages shall be translated.